content/blog/2019/09/2019-09-16-outreachy-audit-log-release.adoc

```diff
@@ -0,0 +1,49 @@
+---
+layout: post
+title: "Audit Log Plugin for Jenkins Releases 1.0"
+tags:
```

Comment:
If you're willing to use an image that I assembled by combining the Jenkins logo and the Apache log4j audit logo, here it is.  Create the images/logos/audit-log directory and place the attached file in that directory.

![Jenkins-audit-log](https://user-images.githubusercontent.com/156685/64999281-67d8ac00-d8a4-11e9-9fc5-24c710c99617.png)

```suggestion
opengraph:
  /images/logos/audit-log/jenkins-audit-log.png
tags:
```